### DISCUSSION TITLE
No tengo ni idea de como empezar el código de condicional para la edad

### DISCUSSION THREAD

**Discussion by Juanaa16** [sebasvelasco353/FDP-2022_01#44] [Q&A]
literalmente no se que hacer o que tipo de if utilizar para desarrollar lo de +50 años o -50 años, ayuda :(

**Comment by sebasvelasco353** [comment]
Realiza este ejercicio:
- Cual es mi objetivo con esto.
- que datos necesito para tomar las decisiones.
- dividir el problema o la accion que quieres en pasos pequenos, para que pase **X** tiene que haber ocurrido **Y**.
- Pensarlo o escribirlo en una forma que vos podas entender y te ayude a conectar esos pequenos pasos, me explico: yo hago una especie de mapa mental con lineas y con datos relevantes que me ayuda a entender la cadena de eventos.
- Escribir el codigo.

Los pasos anteriores te ayudan a entender lo que tenes que hacer y te acostumbra a mantener el orden en el codigo lo cual va a hacer todo (leer, escribir, entender y encontrar codigo) muchisimo mas facil.

Habiendo dicho esto:

**Si la persona tiene 50 años o mas**
- debe activarse un campo donde se informe si se ha realizado o no pruebas de colesterol y triglicéridos en los ultimos 6 meses
- si tiene menos de 50 debe preguntarse si se ha realizado pruebas de Azucar en los ultimos 3 mese.

podes googlearte por ejemplo confirm() o utilizar Events para el cambio en radiobuttons.

**Reply by Juanaa16** [reply]
profe se que la pregunta puede sonar re pendeja pero si quiero diseñarle una imagen de fondo que resolución le pongo en ilustrator? gracias c:

**Reply by sebasvelasco353** [reply]
Hahaha no es una pregunta pendeja, en este caso puedes usar 1280 o 1080, todavía no estamos haciendo responsive entonces con que hagas una HD queda bien.

**Reply by Juanaa16** [reply]
profe, se que es super tarde, pero acaba de pasar algo que me tiene desesperada, intenté subir el repo actualizado con la pagina a github, y no se que carajos hice que reescribió la carpeta y me cambió los archivos por los del DOMINGO :( como los recupero?
